core/src/commonMain/kotlin/com/sourcepoint/mobile_core/models/SPAction.kt

```diff
@@ -13,3 +16,11 @@ enum class SPActionType(val type: Int) {
     IDFADenied(18),
     Unknown(0)
 }
+
+data class SPAction(
+    val type:SPActionType,
+    val campaignType: SPCampaignType,
+    val messageId: String?,
+    val pmPayload: SPJson?,
```

Comment:
Since `SPJson` is now being used not only in network, we should move it from the `network` folder to `models`.